src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.json.in

```diff
@@ -351,5 +358,15 @@
       "kind": "framework",
       "version": "${PackageVersion}"
     },
+    "Microsoft.NETCore.App.Runtime.Windows" : {
+      "kind": "Sdk",
+      "version": "${PackageVersion}",
+      "alias-to": {
+        "win-x64": "Microsoft.NETCore.App.Runtime.win-x64",
+        "win-x86": "Microsoft.NETCore.App.Runtime.win-x86",
+        "win-arm": "Microsoft.NETCore.App.Runtime.win-arm",
+        "win-arm64": "Microsoft.NETCore.App.Runtime.win-arm64"
+      }
```

Comment:
Instead of an alias, should this install them all? I don't know how you deploy a WinUI application, but could you potentially build for `-r win-arm64` from a x64 Windows PC? Is there a way to build for all 4 when creating an app store build?

Reply:
Good question.  I think you're right and we should install them all.